docs/en/stack/monitoring/esms.asciidoc

```diff
@@ -172,3 +164,88 @@ that was provided to you by the Elastic support team.
 If you do not see your metrics yet, see
 <<monitoring-troubleshooting,Troubleshooting {monitor-features}>>.
 --
+
+[discrete]
+[[esms-beats]]
+=== Collecting monitoring data about Beats
+
+:beatname_lc:       packetbeat
+:beatname_uc:       {packetbeat}
+
+There are two methods for sending monitoring data about Beats to {esms-init}.
+You can send it directly to {esms-init} by using {metricbeat} or you can route
+it through exporters on the production cluster.
+
+TIP: It is simplest to use {metricbeat}.
```

Comment:
I would remove this statement because I don't personally feel the statement is true. IMO the old way of collecting metrics is easier to set up right now.

Reply:
That's funny, actually, since I think the new method is easier than setting up HTTP exporters.

Reply:
Yeah, perhaps its best to avoid the value judgement here, unless you think it's particularly helpful?

At some point in the near future we hope to be able to recommend using Metricbeat for collection (over the internal collection method). We could update all Stack Monitoring docs at that time to make the recommendation?

Reply:
That sounds reasonable. I've removed the tips for now.